src/mam4xx/aero_model.hpp

```diff
@@ -867,8 +868,9 @@ void set_f_act_coarse(const int kk,
   f_act_conv_coarse_nacl = 0.80;
 
   // dust and seasalt mass concentration [kg/kg]
-  const int lcoardust = aerosol_index_for_mode(ModeIndex::Coarse, AeroId::DST);
-  const int lcoarnacl = aerosol_index_for_mode(ModeIndex::Coarse, AeroId::NaCl);
+  const int idx_coarse = int(ModeIndex::Coarse);
```

Comment:
In C++ we tend to use static_cast<int>()  instead of the old C style cast int().  I know it is the same result for enums, just a more modern way to do it.